### DISCUSSION TITLE
venus-os_dbus-serialbattery (mr-manuel) vs dbus-serialbattery (Louisvdw)

### DISCUSSION THREAD

**Discussion by BipedalPrimate** [mr-manuel/venus-os_dbus-serialbattery#58] [Q&A] [answered]
Am starting to plan my next upgrade to the dbus-serialbattery and notice that the two repositories (Louisvdw & mr-manuel) have different releases and functionality

I suspect it is quite awkward to maintain two repositories and am wondering if there will be only one repository in the future.

**Comment by mr-manuel** [comment, marked as answer]
I was maintaining the repo of Louis for over 1,5 years now and it is a lot of work. Many people did not understood that I was maintaining and developing the whole driver in this time and they even called me many times Louis. Therefore I decided to not merge the changes anymore and continue only in my repository.

**Comment by BipedalPrimate** [comment]
Is it worthwhile (or allowed in GitHub) to include a note at the start of the Louisvdw readme.md that subsequent to the Louisvdw v1.2.20240408 release that it will no longer be maintained by yourself & future development will be done in the mr-manuel repository?

**Reply by mr-manuel** [reply]
I can‘t do it, since I‘m not the owner of the repository. Only the owner can do that and eventually mark it as public archive. Anyway, if you open the installer you see that my repo is more up to date and the docs are redirecting to my docs.

**Comment by BipedalPrimate** [comment]
OK thanks.
